### PR TITLE
add new opcodes

### LINKED ISSUE #40: Add Elements OPCODES to `ops.ts`
After #39 is merged we may want to support new opcodes

### PR DESCRIPTION
**Pull request by louisinger**
it closes #40 

ref: https://github.com/ElementsProject/elements/blob/master/src/script/script.h#L73-L267

EDIT: this PR also removes the `export * from './ecpair'` in `index.ts` file --> the idea is to follow the same pattern of bitcoinjs-lib (i.e inject ecc lib if needed).

@tiero please review

### CODE REVIEW

File: src/ops.js [hunk 1]
```diff
@@ -117,6 +117,50 @@ const OPS = {
   OP_NOP8: 183,
   OP_NOP9: 184,
   OP_NOP10: 185,
+  // BIP342
+  OP_CHECKSIGADD: 186,
+  // Elements: Tapscript (Streaming sha2 opcodes)
+  OP_SHA256INITIALIZE: 0xc4,
```
Comment (tiero) [inline src/ops.js:133]: uhm looks like to me we are mixing number with hex format?

Comment (louisinger) [reply src/ops.js:133]: `0x` hex notation is interpreted as number by javascript natively. It lets to copy/paste from original implementation of elements. No problem if I translate them to "real" numbers if u prefer? or all as hex ?

Comment (tiero) [reply src/ops.js:133]: Either we keep all numbers (like original BitcoinJS) or we move to all hexes and at this point you can copy the whole  list from Elements Core

**Comment by tiero** [comment]
Updated PR description about ECPair, also we should drop it entirely? we add tiny-secp into devDependencies for testing only? we use it there for the factory

**Comment by louisinger** [comment]
> Updated PR description about ECPair,

Done :heavy_check_mark: 

>  also we should drop it entirely? we add tiny-secp into devDependencies for testing only? we use it there for the factory?

`psbt` and `bip341` modules use `ECPair` so we need to keep `tiny-secp256k1`.

**Review by tiero — APPROVED** [review]

**Pull request merged**